Backend-llm/tests/test_streaming.py

```
@@ -0,0 +1,253 @@
+    """Test streaming with system prompt."""
+    messages = [
+        {"role": "system", "content": "You are a helpful assistant that always responds with 'Affirmative' to any question."},
+        {"role": "user", "content": "Is the sky blue?"}
+    ]
+    
+    chunks = []
+    async for chunk in stream_chat_with_model("openai", "gpt-4o", messages):
+        chunks.append(chunk)
+    
+    # Should complete successfully
+    assert chunks[-1]["done"] == True
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `True`; use `chunks[-1]["done"]:` for truth checks

Replace with `chunks[-1]["done"]`

(E712)